src/java.se/share/data/jdwp/jdwp.spec

```diff
@@ -2018,8 +2019,9 @@ JDWP "Java(tm) Debug Wire Protocol"
         (ErrorSet
             (Error INVALID_THREAD "The thread is null, not a valid thread, or the thread "
                                   "is not alive.")
-            (Error NOT_IMPLEMENTED "The thread is a virtual thread and the target "
-                                  "VM does not support the command on virtual threads.")
+            (Error THREAD_NOT_SUSPENDED "The thread is a virtual thread and was not suspended.")
+            (Error OPAQUE_FRAME   "The thread is a suspended virtual thread and the implementation "
+                                  "was unable to throw an asynchronous exception from this frame.")
```

Comment:
Should it be aligned with JVMTI and some other places in your fix and say
 "from the current frame" instead of "from this frame"?

Reply:
I'm waiting for your JVMTI PR to finish review. I don't want to have to change this more than once.

Reply:
I've done the update.